### PR TITLE
Send zip

### PR DESCRIPTION
**Pull request by sree-cfa**
Send submissions as zipfile through SFTP

### CODE REVIEW

File: src/main/java/org/homeschoolpebt/app/cli/SftpClientImpl.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  public SftpClientImpl(@Value("${sftp.username:}") String username, @Value("${sftp.password:}") String password, @Value("${sftp.upload-url:}") String uploadUrl) {
+    this.username = username;
+    this.password = password;
+    this.uploadUrl = uploadUrl;
+  }
+
+  @Override
+  public void uploadFile(String zipFilename) throws JSchException, SftpException {
+    JSch jsch = new JSch();
+    jsch.setKnownHosts("~/.ssh/known_hosts");
```
Comment (tdooner) [inline src/main/java/org/homeschoolpebt/app/cli/SftpClientImpl.java:30]: Dose this still work even if this file doesn't exist? Maybe we should actually make that file as part of our build process. The value I have for it is:

```
mft.safe.cdt.ca.gov ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCi/KKNiEmAyBT6lmCJ+pzIusbEdYbN7wc3mt6dQz9SDbHRtpNWbRFdsKSyzjNZr3Br92Jl5IxqW1Hjdfa3w66NuyFs/jsyRqrsYZIpn/nxDsPJTXxPCJ2c7Qy34A0O6c2I1w7Imss78Ddc63tlFLaMnL3WuHBRHzt2oneT0rqSn0QFrTUWzz3a3ojMTRksARopyraQ5nJKe1nm1QqbN9NoLZG90aveSACLPJ8RvfMMUTKIW73tFk3UHuOO+6Z7LrEKVuIGLWGuwLK/fyJInjcpKtRV2mwE4dv5UOLGmcBIGk6kU0voZkPUegrG86tVHT7+42AJ0J5gBb4jCVRT7yPV
```

File: src/main/java/org/homeschoolpebt/app/cli/SftpClientImpl.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    jschSession.setPassword(password);
+    jschSession.connect(10000);
+
+    Channel sftp = jschSession.openChannel("sftp");
+    sftp.connect(5000);
+
+    ChannelSftp channelSftp = (ChannelSftp) sftp;
+    channelSftp.put(zipFilename, "/inbox/" + zipFilename);
+
+    channelSftp.exit();
```
Comment (tdooner) [inline src/main/java/org/homeschoolpebt/app/cli/SftpClientImpl.java:41]: Very cool

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -92,6 +92,10 @@ server:
   error:
     whitelabel:
       enabled: false
+sftp:
+  username: ${SFTP_USERNAME}
+  password: ${SFTP_PASSWORD}
+  upload-url: ${SFTP_UPLOAD_URL}
```
Comment (tdooner) [inline src/main/resources/application.yaml:98]: Cool - will you add these to the Github production environment + deploy script, or I can if you'd like?

**Review by tdooner — APPROVED** [review]

File: src/main/resources/application.yaml (hunk 3 above)
Comment (sree-cfa) [reply src/main/resources/application.yaml:98]: Oh yeah, I can add them

**Pull request merged**